WF_Proton_Launcher/Main.cpp

```diff
@@ -580,7 +584,7 @@ void decodeLzmaFile(FILE* src, FILE* dst)
 
 void addClient(CURLM *cm, const Hashlist::RemoteHash& file)
 {
-	std::string url = "http://" + CONTENT_HOST + file.SourceFile;
+	std::string url = "http://" + INDEX_HOST + file.SourceFile;
```

Comment:
Any particular reason why you use the origin domain here? Content seems to be where the actual content is served from after all.

Reply:
origin is their "web content" server. content is what warframe.exe uses. origin is where the official launcher downloads everything before running warframe.exe

(i spent lots  of time figuring this out in wireshark)

Reply:
So it'd make the launcher look more like a launcher to DE and not as a strangely behaving copy of Warframe.

That's okay in that case.